src/random.cpp

```diff
@@ -305,16 +305,14 @@ void GetOSRand(unsigned char *ent32)
             RandFailure();
         }
     }
-#elif defined(HAVE_GETENTROPY) && defined(__OpenBSD__)
```

Comment:
nit: I think you can remove `HAVE_GETENTROPY` now?

Reply:
Good idea, done.